cmd/soroban-cli/src/commands/network/container/shared.rs

```diff
@@ -148,3 +175,14 @@ async fn check_docker_connection(docker: &Docker) -> Result<(), bollard::errors:
         }
     }
 }
+
+pub fn get_container_name(container_name_arg: Option<String>, network: Option<Network>) -> String {
```

Comment:
```suggestion
pub fn get_container_name(container_name_arg: Option<&str>, network: Option<Network>) -> String {
```

Reply:
On second thought, perhaps this method should move into the `impl` since it relies on the internal state.
```rust
  get_container_name(self.container_args.container_name.clone(), self.network)
```
becomes
```rust
  self.container_args.name(self.network)
```

Reply:
Also generally it's better to have `Option<&T>` an the argument type because you can always do `x.as_ref` or `x.as_deref`, and need to have ownership.

Reply:
That makes sense to move `get_container_name` into the impl of `Args` - I pushed up a change for that.